Library/Homebrew/shims/scm/git

```diff
@@ -28,9 +34,7 @@ realpath() {
     else
       path="$dir/$dest"
     fi
-    dir="$(quiet_safe_cd "${path%/*}/" && pwd -P)"
-    base="${path##*/}"
-    path="$dir/$base"
+    path="$(abspath "$path")"
```

Comment:
Kindly remind you that `$dir` will be used in the while block for next iteration. As result, you cannot use `abspath` directly like this.